pages/[post].tsx

```diff
@@ -1,26 +1,26 @@
 import React from "react";
 import { useRouter } from "next/router";
-import { postData } from "../utils/post-data";
+import postData from "../utils/post-data.json";
 import { Heading, Image, Text, Flex, Box } from "@chakra-ui/react";
 
 export default function PostOne() {
```

Comment:
`PostOne` means `Primeiro Post` on the other hand `OnePost` means `Um Post (um post qualquer)`